src/main/java/net/runelite/client/plugins/microbot/lunarbuckets/LunarBucketsPlugin.java

```diff
@@ -0,0 +1,131 @@
+package net.runelite.client.plugins.microbot.lunarbuckets;
+
+import com.google.inject.Provides;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.plugins.microbot.PluginConstants;
+import net.runelite.client.plugins.microbot.util.antiban.Rs2AntibanSettings;
+import net.runelite.client.plugins.microbot.util.antiban.Rs2Antiban;
+import net.runelite.client.plugins.microbot.util.antiban.enums.Activity;
+import net.runelite.client.plugins.microbot.util.grandexchange.Rs2GrandExchange;
+import net.runelite.client.ui.overlay.OverlayManager;
+import net.runelite.api.gameval.ItemID;
+
+import java.time.Duration;
+import java.time.Instant;
+
+import javax.inject.Inject;
+
+@Slf4j
+@PluginDescriptor(
+        name = LunarBucketsConstants.NETO + "Lunar Buckets",
+        description = "Casts Humidify to fill buckets of water",
+        tags = {"lunar", "humidify", "magic", "bucket", "MoneyMaking"},
+        authors = {"Neoxic"},
+		minClientVersion = "1.9.9.2",
```

Comment:
```suggestion
		minClientVersion = "1.9.9.2",
		iconUrl = "https://chsami.github.io/Microbot-Hub/LunarBucketsPlugin/assets/icon.png",
        cardUrl = "https://chsami.github.io/Microbot-Hub/LunarBucketsPlugin/assets/card.png",
```

We cannot forget about images right 😄